vuejs-cli/README.md

```diff
@@ -0,0 +1,88 @@
+# Vue.js debugging in VS Code with Chrome and vue-cli
```

Comment:
I'd say more like `in VS Code and Chrome`. Don't feel vue-cli needs to get mentioned in the title.

Reply:
Also, maybe show some synchronized stepping or show that you can also inspect / F10 in Chrome (which is cool).